Meridian59/Data/DataController.cs

```diff
@@ -1902,6 +1913,52 @@ protected void HandleMessage(MessageMessage Message)
             ChatMessages.Add(Message.Message);           
         }
 
+        public void HandleAdminMessage(AdminMessage Message)
```

Comment:
This should be 'protected' like the other internal handlers, because it is only called from class internal *HandleIncomingGameModeMessage(GameMessage Message)*